src/ApiStack.ts

```diff
@@ -224,6 +236,29 @@ export class ApiStack extends Stack {
     return persoonsGegevensFunction;
   }
 
+  private uitkeringenFunction(baseUrl: string, readOnlyRole: Role, mtlsConfig: TLSConfig) {
+    const uitkeringenFunction = new ApiFunction(this, 'uitkeringen-function', {
+      description: 'Uitkeringen-lambda voor de Mijn Nijmegen-applicatie.',
+      codePath: 'app/uitkeringen',
+      table: this.sessionsTable,
+      tablePermissions: 'ReadWrite',
+      applicationUrlBase: baseUrl,
+      readOnlyRole,
+      environment: {
+        MTLS_PRIVATE_KEY_ARN: mtlsConfig.privateKey.secretArn,
+        MTLS_CLIENT_CERT_NAME: Statics.ssmMTLSClientCert,
```

Comment:
Waarom haal je de private key op uit de mltsConfig en de certname niet?
```
  private mtlsConfig() {
    const privateKey = aws_secretsmanager.Secret.fromSecretNameV2(this, 'tls-key-secret', Statics.secretMTLSPrivateKey);
    const clientCert = StringParameter.fromStringParameterName(this, 'tlskey', Statics.ssmMTLSClientCert);
    const rootCert = StringParameter.fromStringParameterName(this, 'tlsrootca', Statics.ssmMTLSRootCA);
    const tlsConfig = { privateKey, clientCert, rootCert };
    return tlsConfig;
  }
```

Reply:
Voor de param heb ik alleen de naam van de parameter nodig, waarbij de static waarde (die toch beschikbaar is) genoeg is. De hele config gaat mee omdat voor het geven van permissies wel nodig is. Ik zou het ook met iets als `mtlsConfig.clientCert.parameterName` kunnen doen, dat is wel duidelijker dan een soort van magische constante. Neem ik mee als verbeter/verduidelijkpunt.

Reply:
Ik had al het vermoeden dat het een refactor was, het is dus ook echt niet blocking, maar wel opvallend aangezien dit de focus is van de PR